sqlalchemy_ibmi/base.py

```diff
@@ -762,6 +763,17 @@ def _get_default_schema_name(self, connection):
             default_schema_name = default_schema_name.strip()
         return self.normalize_name(default_schema_name)
 
+    # Driver version for IBM i Access ODBC Driver is given as
+    # VV.RR.MMSP where VV (major), RR (release), and MM (modificaiton level)
+    # are all two digits, and the SP (service pack level) can be 0, 1, 2, or
+    # 3 digits
+    def get_driver_version(self, db_conn):
+        version = db_conn.getinfo(self.dbapi.SQL_DRIVER_VER).split('.')
+        mmsp = version.pop(2)
+        mm = mmsp[:2]
+        sp = mmsp[2:] if mmsp[2:] else '0'
+        return [int(_) for _ in version.extend([mm, sp])]
```

Comment:
```suggestion
        version.extend([mm, sp])]
        return [int(_) for _ in version]
```
list.extend does not return the the list object, but instead returns None. You'll need to pull that out